### DISCUSSION TITLE
Win7-64: Zoom and Cursor Measurements

### DISCUSSION THREAD

**Discussion by MaMiKoe** [OpenHantek/OpenHantek6022#264] [Q&A] [answered]
openhantek_20211216-31c5ee2-1_win_x64:
I dont know whether it is a malfunction, depending on Win7 oder OpenHantek, because I can not test it with Linux (I am a Linux-DAU) for compare:
In the Zoom-Window it is not possible to make "Cursor Measurements". After selecting CH1 or CH2 an setting to ON I can draw the rectangle in the Zoom-window and in the "normal" Windows above the corresponding rectangle is also visible. But this has no effect to the measurements (the values), which are displayed in the "Cursor Measurements"-window. Only drawing the rectangle directly in the "normal" window results in (new) measurement-values. I assume this is a malfunction in Win.
And: Also when setting CH1 or CH2 to OFF "Cursor Measurements" are made (in the above, "normal" window) and a rectangle is drawn "invisible". So when setting CH1 or CH2 again to "ON" the before "invisible" drawn rectangle becomes visible. If this behaviour does also appear in the Linux-version I would like to suggest to change this: IMHO it would be advantegous, if the visible drawn rectangle and its values would be kept while the channel is set to OFF and drawing an "invisible" rectangle while this OFF-state would only result in displaying the measurements while "invisible" drawing and until the mouse-button is released, and the measurement of the kept "visible" drawn rectangle are displayed again.

**Comment by Ho-Ro** [comment, marked as answer]
The whole measurement function is a kind of *terra incognita* for me, it was invented a long time ago and I haven't touched it since I started maintaining the OH6022. To be honest, I don't use it and therefore it is a low priority for me. But anyway - if something is drawn on the screen, the values displayed should still match. Thank you for your detailed report - I was able to reproduce the errors on Linux (Debian stable)  and will investigate further in the source code.

**Reply by Ho-Ro** [reply]
Fixed with latest unstable.
